.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,14 @@
 
   Also mention if you think this PR needs to be backported. And list relevant or fixed issues.
 
+------------------------
+
+## Policies
+
+- "Draft" and "WIP" pull requests are **not** accepted on this repository. Please get your work to a stage where **you** consider it ready for
+merge prior to opening a pull request. The one exception to this rule is opening a short term, not-for-merge PR in order to conduct a full CI test run on the branch. Please make sure you close the pull request ASAP after conducting this test in order to keep the QGIS pull request queue clean and manageable.
```

Comment:
> @troopa81 are you sure? We have particular conditions for jobs running under QGIS GH account, if the same workflow is run in the user's account it will certainly fail (timeouts).

It was still working a few weeks ago. It's my usual workflow when I want to try a modification that have great chance to break things (this [one](https://github.com/troopa81/QGIS/actions/runs/7278417082) was all green for instance)

> That means running this on your own fork which comes with additional configuration effort (which might hard to find for people not used to the github ecosystem)?

Only this one no?

![actions](https://github.com/qgis/QGIS/assets/14358135/0dd4a59d-0332-446e-90db-fa6025263b6c)

Reply:
If you know which buttons to press and files to change it's "easy" (but there's still the cold cache which might time out according to @elpaso)
But I am not convinced the "win" for us is big enough to justify raising the entry barrier with this and the extra effort for the review team to also monitor a mailing list.